wurst/util/UnitUtils.wurst

```diff
@@ -0,0 +1,60 @@
+package UnitUtils
+import BuffObjEditing
+import ClosureTimers
+
+
+
+/*	This file contains various utility functions for Unit, 
+	which require enhanced functionality like closures and cannot be defined in Unit.wurst */
+
+@configurable public real FIELD_OF_VIEW_FRONT  = 230.00
+@configurable public real FIELD_OF_VIEW_BEHIND = 115.00  // Real FoV is: 180 - 2x (FIELD_OF_VIEW-90)	Using a value of 115 generates a FoV of 130   (natural FoV is about 135 degrees)
+
+/** Apply Buff to this unit for duration.*/
+public function unit.applyBuffTimed(buffTuple buffApplied, real duration)
+	this.addAbility(buffApplied.abilId)
+	this.setAbilityLevel(buffApplied.abilId, 1)
+	doAfter(duration) ->
+		if this != null
+			this.setAbilityLevel(buffApplied.abilId, 0)
+			this.removeAbility(buffApplied.abilId)
+			
+/** Increase Armor of a unit for a duration */
+public function unit.addArmorTimed(real armorAdd, real duration)
+	BlzSetUnitArmor(this, this.getArmor()+armorAdd)
+	doAfter(duration) ->
+		if this != null
+			BlzSetUnitArmor(this, this.getArmor()-armorAdd)
```

Comment:
This is not safe to multi-instanceability and would need a lock or guard

Reply:
What about making these functions @configurable ?
Anyone could change these based on their needs. I'd say we hardly can cover all the possible cases inside Stdlib.

Reply:
I'm not a fan, but will defer to frotty. Prefer this in library

Reply:
removed